doc/rel_notes/1.0.0.txt

```diff
@@ -29,7 +29,10 @@ Build System
 
  * Simplified string logic for finding compiler flags file.
 
- * Added Eigen3 include directory path to include directories exported by MAP-Tk
+ * Added Eigen3 include directory path to include directories exported by
+   MAP-Tk.
+
+ * Added support for LIB_SUFFIX.
```

Comment:
Apologies; I was under the impression that the `MAPTK_LIB_SUFFIX` support was not present in the previous release. Since it appears it was, I agree that the note should reflect that.